app/views/catalog/show.html.erb

```diff
@@ -3,6 +3,13 @@
 
     <%= render "shared/record/record_toolbar" %>
 
+    <!-- TODO: remove after migration is complete -->
+    <% if folio_disclaimer(@document) %>
+      <div class="alert alert-warning text-center" role="alert">
+       <b>MIGRATION ERROR</b>: FOLIO is missing source record data; this bibliographic data was derived from the FOLIO instance metadata.
```

Comment:
tiny thing, but could we throw an icon of some sort in here? maybe a "caution" or "danger" indicator? i think it'd help draw the eye to it even more.

Reply:
How's this? If it looks OK I will push up the code.
![Screenshot 2023-06-21 at 14 46 47](https://github.com/sul-dlss/SearchWorks/assets/1328900/bb69a8cb-4883-4e41-8715-b987c73feb6e)

Reply:
I could even put them on both sides of the text haha

Reply:
I pushed up the icon.

Reply:
looks great, thanks!